packages/gitea/build.sh

```diff
@@ -21,6 +21,7 @@ termux_step_make() {
 	cp -a "$TERMUX_PKG_SRCDIR" "$GOPATH"/src/code.gitea.io/gitea
 	cd "$GOPATH"/src/code.gitea.io/gitea
 
+	unset CGO_LDFLAGS
```

Comment:
Removing LDFLAGS will remove the runpath from gitea binary. The following output is present in current gitea version 1.21.5 but it is absent after this change.

```
$ readelf -d $PREFIX/bin/gitea | grep runpath
  0x000000000000001d (RUNPATH)         Library runpath: [/data/data/com.termux/files/usr/lib:/data/data/com.termux/files/usr/lib:/data/data/com.termux/files/usr/lib:/data/data/com.termux/files/usr/lib:/home/builder/.termux-build/_cache/android-r26b-api-24-v1/lib]
```